libraries/chain/chain_controller.cpp

```diff
@@ -368,7 +369,8 @@ transaction_trace chain_controller::delayed_transaction_processing( const transa
    time_point_sec execute_after = head_block_time();
    execute_after += mtrx.delay;
    //TODO: !!! WHO GETS BILLED TO STORE THE DELAYED TX?
-   deferred_transaction dtrx(context.get_next_sender_id(), config::system_account_name, config::system_account_name, execute_after, trx);
+   fc::uint128_t _id(trx.id()._hash[3], trx.id()._hash[2]);
```

Comment:
If the decision is to stay with uint128_t, then we should make this into a method, rather than maintaining this pattern in 3 or more places, or why don't we just have the user use the deferred_transaction::sender_id, and just do the conversion when we construct the deferred_transaction?

Reply:
So, the changes I have are for generated deferred transactions and not user produced deferred.  I like changing this just a function,  it just seems more natural for the user to just give a transaction id for the transaction they had originally sent.